cmd/wnode-status/config.go

```diff
@@ -68,15 +68,20 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 	whisperConfig.MinimumPoW = *minPow
 	whisperConfig.TTL = *ttl
 
-	if whisperConfig.EnableMailServer && whisperConfig.PasswordFile == "" {
-		return nil, errors.New("mail server requires -password to be specified")
+	if whisperConfig.EnableMailServer && whisperConfig.Password == "" && whisperConfig.PasswordFile == "" {
+		return nil, errors.New("either a password file or a password should be specified for mail server")
+	}
+
+	if whisperConfig.PasswordFile == "" {
```

Comment:
As I mentioned, this is not needed. We can acknowledge using a single password but it does not have to be hardcoded in the code itself.